content/en/continuous_integration/pipelines/custom_commands.md

```diff
@@ -137,6 +137,12 @@ the `DD_GITHUB_JOB_NAME` environment variable needs to be exposed, pointing to t
         - run: datadog-ci trace ...
     ```
 
+## Troubleshooting
+
+### Payload too large
+The size limit is approximately `4MB`. The most common cause for this error are extremely large tags.
```

Comment:
Is this the exact error that customers will receive?

Reply:
Yep the error message is `payload too large` so if they Ctrl+F the error in the page they can get there